server/middlewares/frontendMiddleware.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const path = require('path');
+
+// Dev middleware
+const addDevMiddlewares = (app, options) => {
+  const webpackDevMiddleware = require('webpack-dev-middleware');
+  const webpackHotMiddleware = require('webpack-hot-middleware');
```

Comment:
Yeah, that's true. Mainly because we require webpack.prod.babel.js file in index.js. We need to extract common config of webpack to a file that does not have any requires, so we don't have to install it in production (e.g. publicPath). That's one of the possibilities.